lib/handlers/injector-sor.ts

```diff
@@ -71,7 +74,7 @@ export interface RequestInjected<Router> extends BaseRInj {
 }
 
 export type ContainerDependencies = {
-  provider: ethers.providers.JsonRpcProvider
+  provider: ethers.providers.BaseProvider
```

Comment:
I was thinking to make the provider type as generic as possible, as opposed to being concrete. However, in this case `JsonRPCProvider` makes more sense, since we do have the north star view of having some consolidated Json RPC Clients.